app/services/foreman_rh_cloud/cloud_request_forwarder.rb

```diff
@@ -0,0 +1,75 @@
+require 'rest-client'
+
+module ForemanRhCloud
+  class CloudRequestForwarder
+    include ::ForemanRhCloud::CloudAuth
+
+    def forward_request(original_request, controller_name, branch_id)
+      forward_params = prepare_forward_params(original_request, branch_id)
+      Rails.logger.debug{"Request parameters for telemetry request: #{forward_params}"}
+
+      forward_payload = prepare_forward_payload(original_request, controller_name)
+
+      Rails.logger.debug("User agent for telemetry is: #{http_user_agent original_request}")
```

Comment:
How useful is logging user agent?

Reply:
It depends what information is sent in the user agent by insihgts-client. From looking at the code in `prepare_forward_params`, the user agent actually modifies the parameters of the outgoing request.